portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -28,21 +31,41 @@
 public class DataServlet extends HttpServlet {
 
     private List<String> commentList = new ArrayList<>();
+    private Entity commentEntity = new Entity("Comment");
     private Gson gson = new Gson();
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-      
-    initializeComments();
     String json = convertToJsonUsingGson(commentList);
     response.setContentType("application/json;");
     response.getWriter().println(json);
   }
 
-  private void initializeComments() {
-    commentList.add("Awesome Website!");
-    commentList.add("Great job! Keep it up!");
-    commentList.add("Very interesting!!");
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+      String comment = getParameter(request, "comment-input", "");
+      if(isValid(comment)) {
```

Comment:
come to think of it, can we do some input validation in the UI? That way we have a more responsive web app

Reply:
Sounds good! I'll display a little message prompting the user to try again